### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by HarryStevens** [dr-skot/kenken#2]
I wanted to play around with your code, but before doing so, I was curious to know what the license is. Please let me know.

**Comment by dr-skot** [comment]
Hi. Thanks for asking. GPL3 is my preference, will that suit your purposes?

**Comment by HarryStevens** [comment]
Absolutely. Sorry for the late reply; I thought GitHub would send me an email.
